.github/workflows/scripts/spdx-tools-java-wrapper.sh

```diff
@@ -8,42 +8,54 @@
 set -euo pipefail
 
 version="1.1.4"
-zip="tools-java-${version}.zip"
-url="https://github.com/spdx/tools-java/releases/download/v${version}/${zip}"
 jar="$HOME/spdx-tools-java/tools-java-${version}-jar-with-dependencies.jar"
 
-if [[ -n "$JAVA_HOME" && -x "$JAVA_HOME/bin/java" ]]; then
-    javabin="$JAVA_HOME/bin/java"
-else
-    javabin=java
-fi
+spdx_tools() {
```

Comment:
Unfortunately, the commit message does not detail the very different changes done in this commit.